packages/esm-patient-chart-app/src/workspace/context-workspace.component.tsx

```diff
@@ -10,30 +12,83 @@ import { useWorkspace } from '../hooks/useWorkspace';
 import { patientChartWorkspaceSlot } from '../constants';
 import { isDesktop } from '../utils';
 
+type WindowSize = 'maximize' | 'normal' | 'hide';
+type ActionTypes = 'minimize' | 'maximize' | 'hide' | 'reopen';
+
+const reducer = (state: WindowSize, action: ActionTypes) => {
+  switch (action) {
+    case 'maximize':
+      return (state = 'maximize');
```

Comment:
Hm this assignment is rather confusing. Why not just `return 'maximize'`? Same for the rest.